### ISSUE TITLE
Null values are causing problems

### ISSUE THREAD

**Issue by rosencreuz** [MicroUtils/kotlin-logging#16]
`LocationAwareKLogger` class uses not nullable arguments for logging operations. Because of this when a null parameter value is passed it causes an exception and doesn't log anything. Passing such values are OK according to slf4j api and it works fine with underlying implementations (log4j, logback, etc.)

```
val param : String? = null
logger.warn("Nulls should work {}!", param)
```

> Caused by: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method mu.internal.LocationAwareKLogger.warn, parameter arg
>    at mu.internal.LocationAwareKLogger.warn(LocationAwareKLogger.kt)

**Comment by oshai** [comment]
what version are you using?

**Comment by oshai** [comment]
because the issue was fixed in 1.4.2: https://github.com/MicroUtils/kotlin-logging/pull/14

**Comment by rosencreuz** [comment]
I was using 1.4.1. It works with 1.4.2. Thanks!
